Update readme.md

mrhappyma opened the pull request.

mrhappyma commented:
@techyporcupine ?

techyporcupine commented:
Please remove `  - And for some reason [techyporcupine](https://github.com/techyporcupine) put ads on the site... I'm pretty sure he also was the one to right to email to [barak](https://www.youtube.com/c/CRACKLEgames).` that is incorrect. I did not put ads on it, and i do not understand tha barak thing

mrhappyma commented:
Wasn't the point of it to email barak about a "sponsorship" and rickroll him?
@techyporcupine

techyporcupine commented:
> Wasn't the point of it to email barak about a "sponsorship" and rickroll him? @techyporcupine

I do not believe so... I thought it was just to make a lightweight rick site.

techyporcupine commented:
> Wasn't the point of it to email barak about a "sponsorship" and rickroll him? @techyporcupine

And @piethoncode wanted to make a site with the name of termitefarms

mrhappyma commented:
> I do not believe so... I thought it was just to make a lightweight rick site.

Alrighty i guess i can remove that then

and i implemented rick you stole my stuff from my other thing before i was done...
dont really like the wording of your version...
thats also not much of an accomplishment you just upload the file 🤷‍♂️

techyporcupine commented:
> and i implemented rick you stole my stuff from my other thing before i was done... dont really like the wording of your version... thats also not much of an accomplishment you just upload the file 🤷‍♂️

I modified it to not include toothpaste stuff.

mrhappyma commented:
> > and i implemented rick you stole my stuff from my other thing before i was done... dont really like the wording of your version... thats also not much of an accomplishment you just upload the file 🤷‍♂️
> 
> I modified it to not include toothpaste stuff.

i had a branch i was about to open a pr for with everything done and i had quizbowl
then you took the toothpaste stuff and essentially re-created my pr

mrhappyma commented:
you originally just had youtube's player on there, looking in the commit history

mrhappyma commented:
nvm
